lib/webhooks/webhook_services.ex

```diff
@@ -10,6 +10,8 @@ defmodule ApplicationRunner.Webhooks.WebhookServices do
   alias ApplicationRunner.Webhooks.Webhook
 
   def create(env_id, params) do
+    Webhook.new(env_id, params)
```

Comment:
I think the best way to create an Env/User webhook is to call create with two parameters,
Token params and body params, so you can do two create functions
create(%{env : env_id}}, body_params)
create(%{env : env_id, user : user_id}}, body_params)

with Token params the result of Guardian.Plug.current_resource